src/test/java/uk/gov/justice/hmpps/prison/service/receiveandtransfer/PrisonerTransferServiceTest.java

```diff
@@ -30,13 +31,13 @@
 @ContextConfiguration(classes = TestClock.class)
 @ActiveProfiles("test")
 @Transactional
-public class PrisonerReleaseAndTransferServiceTest {
+public class PrisonerTransferServiceTest {
```

Comment:
Another test we possibly don't need, probably covered by resource test, but for now point at refactored service since it behaves the same